EmailSender/src/main/java/com/exadel/EmailSender/service/UserService.java

```diff
@@ -24,4 +24,13 @@ public List<UserDto> getAllUsers() {
                 .collectList()
                 .block();
     }
+
+    public UserDto getUser(Long id){
+        return webClientBuilder.build()
+                .get()
+                .uri("http://localhost:8080/user/getUser/" + id)

```

Comment:
Better create some class which will be contains all fields related to our main service